examples/player/simid_player.js

```diff
@@ -197,33 +220,100 @@ class SimidPlayer {
   }
 
   /**
-   * Returns the dimensions of an element within the player div.
+   * Returns the full dimensions of an element within the player div.
+   * These dimensions will equal the full size of the player.
    * @return {!Object}
    */
-  getDimensions(elem) {
-    // The player div wraps all elements and is used as the offset.
-    const playerDiv = document.getElementById('player_div');
-    const playerRect = playerDiv.getBoundingClientRect();
+  getFullVideoDimensions(elem) {
     const videoRect = elem.getBoundingClientRect();
+
     return {
-      'x' : videoRect.x - playerRect.x,
-      'y' : videoRect.y - playerRect.y,
+      'x' : 0,
+      'y' : 0,
       'width' : videoRect.width,
       'height' : videoRect.height,
       // TODO: This example does not currently support transition duration.
       'transitionDuration': 0
     };
   }
 
+  /**
+   * Returns the specified dimensions of the non-linear creative.
+   * @return {!Object}
+   */
+  getNonLinearDimensions() {
```

Comment:
This function says its returing the dimensions of the non-linear creative, but its taking the values from the input buttons.  If you initialize the nonlinear creative, but then the creative requests a change in size, then you call this again, it will return the wrong values

Reply:
this function returns the SPECIFIED dimensions meaning the original ones that were declared from initialization. Maybe I just need to make the comment clearer